tests/test_glue/test_schema_registry.py

```diff
@@ -9,57 +9,65 @@
 
 from . import helpers
 from .fixtures.schema_registry import (
-    REGISTRY_NAME,
-    REGISTRY_ARN,
-    DESCRIPTION,
-    TAGS,
-    SCHEMA_NAME,
-    SCHEMA_ARN,
-    AVRO_SCHEMA_DEFINITION,
-    NEW_AVRO_SCHEMA_DEFINITION,
-    JSON_SCHEMA_DEFINITION,
-    NEW_JSON_SCHEMA_DEFINITION,
-    PROTOBUF_SCHEMA_DEFINITION,
-    NEW_PROTOBUF_SCHEMA_DEFINITION,
-    AVRO_DATA_FORMAT,
-    JSON_DATA_FORMAT,
-    PROTOBUF_DATA_FORMAT,
-    REGISTRY_ID,
-    SCHEMA_ID,
-    BACKWARD_COMPATIBILITY,
-    DISABLED_COMPATIBILITY,
-    AVAILABLE_STATUS,
+    TEST_REGISTRY_NAME,
+    TEST_REGISTRY_ARN,
+    TEST_DESCRIPTION,
+    TEST_TAGS,
+    TEST_SCHEMA_NAME,
+    TEST_SCHEMA_ARN,
+    TEST_AVRO_SCHEMA_DEFINITION,
+    TEST_NEW_AVRO_SCHEMA_DEFINITION,
+    TEST_JSON_SCHEMA_DEFINITION,
+    TEST_NEW_JSON_SCHEMA_DEFINITION,
+    TEST_PROTOBUF_SCHEMA_DEFINITION,
+    TEST_NEW_PROTOBUF_SCHEMA_DEFINITION,
+    TEST_AVRO_DATA_FORMAT,
+    TEST_JSON_DATA_FORMAT,
+    TEST_PROTOBUF_DATA_FORMAT,
+    TEST_REGISTRY_ID,
+    TEST_SCHEMA_ID,
+    TEST_BACKWARD_COMPATIBILITY,
+    TEST_DISABLED_COMPATIBILITY,
+    TEST_AVAILABLE_STATUS,
+    TEST_SCHEMA_VERSION_NUMBER,
+    TEST_SCHEMA_VERSION_NUMBER_LATEST_VERSION,
+    TEST_VERSION_ID,
+    TEST_INVALID_SCHEMA_NAME_DOES_NOT_EXIST,
+    TEST_INVALID_SCHEMA_ID_SCHEMA_DOES_NOT_EXIST,
+    TEST_INVALID_SCHEMA_ID_REGISTRY_DOES_NOT_EXIST,
+    TEST_METADATA_KEY_VALUE,
+    TEST_METADATA_KEY,
+    TEST_METADATA_VALUE,
+    TEST_DELETING_STATUS,
 )
 
 
-def create_glue_client():
+@pytest.fixture
+def client():
     return boto3.client("glue", region_name="us-east-1")
```

Comment:
With this you could move all the `mock_glue` decorators in this fixture:
```suggestion
    with mock_glue():
        yield boto3.client("glue", region_name="us-east-1")
```

This will make sure that all tests that use the client through this fixture have mocking applied. We could then remove the `@mock_glue` from all tests and can't forget it in future tests :)

Reply:
Agreed - with so many tests, it is always nice to reduce repetition

Reply:
Got it:)